test/test_package.py

```diff
@@ -24,6 +26,139 @@ class TestPackage(MFPackage):
     a = MFArray(block="packagedata", description="array", shape=(3))
 
 
+class TestGwfIc(MFPackage):
+    __test__ = False  # tell pytest not to collect
+
+    export_array_ascii = MFKeyword(
+        block="options",
+        name="export_array_ascii",
```

Comment:
`name` gets set to the attribute name in `MFPackageMeta.__new__` so there is no need to set it. thinking maybe we should disallow setting it explicitly when used within a block or package so there can be no conflict..